templates/contest/list.html

```diff
@@ -255,7 +259,6 @@ <h4>{{ _('Upcoming Contests') }}</h4>
             <br>
         {% endif %}
         <br>
-
```

Comment:
This shouldn't be removed, it's there to separate the current, upcoming, and past contests.